packages/react-components/src/templates/SharedResearchOutput.tsx

```diff
@@ -162,7 +171,33 @@ const SharedResearchOutput: React.FC<SharedResearchOutputProps> = ({
             }`}
             onSave={() => toggleReview(!reviewRequestedBy)}
             onCancel={() => {
-              setDisplayModal(false);
+              setDisplayReviewModal(false);
+            }}
+          />
+        )}
+        {displayPublishModal && (
```

Comment:
I know i started this pattern but if we get any more modals we really need to make something more generic than a flag for each modal